### DISCUSSION TITLE
Copilot

### DISCUSSION THREAD

**Discussion by Enlight432** [immersive-translate/immersive-translate#2109] [Ideas]
### New Feature

1.Add Copilot to the list of services

### Why you want to add these features

1. It is free.
2. It has no limitations.
3. It uses unlimited GPT-4o mini

**Comment by wilsonmilson** [comment]
Copilot comes with prompts, which are not suitable for translation guidance words. The context is extremely limited, the generation speed is slow, and it is likely to encounter excessive requests when used for translation
